otter/metrics.py

```diff
@@ -253,6 +254,40 @@ def connect_cass_servers(reactor, config):
         seed_endpoints, config['keyspace'], disconnect_on_cancel=True)
 
 
+def log_divergent_groups(clock, divergent_groups, log, timeout, group_metrics):
+    """
+    Log groups that have not changed and been divergent for long time
```

Comment:
On the other hand, I think it will be better to return updated copy of `divergent_groups` instead of mutating it. That way it remains pure. I'll do that.